From 6aebd80b87cdc42ba4871bd8284556b5f195f094 Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Sun, 1 Feb 2015 21:10:41 +0900
Subject: [PATCH] cubestorm: enable double buffer on linux

Like noof, at least on Fedora 22 x86_64
while cubestorm shows some graphics on xscreensaver-demo
window, on full screen saver mode cubestorm
shows just a blank screen, like as written
on launchpad #1390304.

Enabling double buffering seems to make
cubestorm show graphics on save mode like
noof.
---
 hacks/glx/cubestorm.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/hacks/glx/cubestorm.c b/hacks/glx/cubestorm.c
index f3fd88b..da0ae53 100644
--- a/hacks/glx/cubestorm.c
+++ b/hacks/glx/cubestorm.c
@@ -229,6 +229,9 @@ init_cube (ModeInfo *mi)
 # ifdef HAVE_JWZGLES
   dbuf_p = True;
 # endif
+# ifdef __linux__
+  dbuf_p = True;
+# endif
 
   bp = &bps[MI_SCREEN(mi)];
 
-- 
2.2.2

